### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by kirasok** [jotaen/klog#264] [Show and tell]
Hi! I like klog very much so I've made an [AUR package](https://aur.archlinux.org/packages/klog-time-tracker-bin) for it.  
Would you add it as one of the ways to install klog?

**Comment by jotaen** [comment]
Cool, glad to hear that you like klog, and thanks for setting this up!

I’ll think about how to link to “external” install packages (i.e., ones that are maintained by other people). Meanwhile, I have a question: is the AUR package automatically updated on every klog release, or is that done manually?

**Reply by kirasok** [reply]
It is manual. Maintainer updates PKGBUILD to reflect changes that would be needed for installation: version, hash sum, dependencies etc and generates new .SRCINFO from updated PKGBUILD. Then maintainer commits changes to the git repository on [aur.archlinux.org](aur.archlinux.org) and new version of package becomes available for every user.

[PKGBUILD of klog](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=klog-time-tracker-bin) is simple. It has variables `pkgname` and `pkgver` to define name and version. Then it fetches from `source=($pkgname-$pkgver.zip::https://github.com/jotaen/klog/releases/download/$pkgver/klog-linux.zip)`, checks for `sha256sum` and installs `LICENSE`, `klog` to `usr`

**Reply by jotaen** [reply]
I’ve linked the AUR package from the [klog website](https://klog.jotaen.net/#get-klog)!

**Comment by rafalkrupinski** [comment]
Thank you! :)

**Comment by alerque** [comment]
BTW, this thread is pretty old. I've since setup a build-from-source version of the AUR package [here](https://aur.archlinux.org/packages/klog-time-tracker) that may be preferable for some users as opposed to the one mentioned above that is a -bin package that wraps the upstream generated binary only.
